cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js

```diff
@@ -70,6 +70,8 @@ describe('ui-data-import', () => {
     // TODO delete function after fix https://issues.folio.org/browse/MODDATAIMP-691
     DataImport.verifyUploadState();
     DataImport.uploadFile('ediFileForC347615.edi', fileName);
+    // TODO need to wait until big file is uploaded
+    cy.wait(5000);
```

Comment:
Maybe beter would be put waiter itno the method?)

Reply:
Done